enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/SøknadEttersedning.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.familie.kontrakter.ef.søknad
+
+data class SøknadEttersending(  val personalia: Søknadsfelt<Personalia>,
+                                val innsendingsdetaljer: Søknadsfelt<Innsendingsdetaljer>
+                            )
```

Comment:
Hvordan skal denne brukes?

Trenger vel fort noen flere felter?

Reply:
Den skal brukes i ettersendingMapper for å lage en "SøknadMedVedlegg" som skal sendes videre til mottak. Søknad med vedlegg tar inn relevant data (som for oss defineres i SøknadEttersending) og vedlegg (filene som lastes opp). Slik jeg har skjønt det skal vi ha nok info til å identifisere hvem som har lastet opp gjennom personalia variabelen, og infomasjon om innsendingen via innsendingsdetaljer variabelen.  Mulig vi trenger mer data for å gjøre noe annet/mer?